apps/supernova/src/App.jsx

```diff
@@ -38,8 +39,9 @@ function App(props = {}) {
     <MessagesProvider>
       <CustomAppShell>
         <ErrorBoundary fallbackRender={fallbackRender}>
```

Comment:
[next PR]: consider moving up the `ErrorBoundary` to catch errors within the CustomAppShell and MessagesProvider

Reply:
resolved